src/backend/expungeservice/expunger/expunger.py

```diff
@@ -64,7 +64,9 @@ def _filter_cases_with_errors(charges: List[Charge]):
             if not charge.skip_analysis():
                 case_number = charge.case()().case_number
                 if not charge.disposition:
-                    cases_with_missing_disposition.add(case_number)
+                    if charge.case()().closed():
```

Comment:
```
if not charge.disposition and charge.case()().closed():
```

Reply:
NayIf there is a charge disposition, line 70 should run. If this condition checked jointly, it's possible for there to be  charge disposition but line 70 doesn't run. I tried writing it as an `and` expression and it caused some unrelated test to fail.

Reply:
`elif charge.disposition and charge.disposition.status == DispositionStatus.UNRECOGNIZED`

Reply:
```
if not charge.disposition and charge.case()().closed():
  ...
elif charge.disposition and charge.disposition.status == DispositionStatus.UNRECOGNIZED:
  ...
```

Reply:
Ok sounds good